src/otx/recipe/semantic_segmentation/dino_v2.yaml

```diff
@@ -2,10 +2,10 @@ model:
   class_path: otx.algo.segmentation.dino_v2_seg.DinoV2Seg
   init_args:
     label_info: 2
-    model_version: dinov2_vits14
+    model_version: dinov2-small-seg
     input_size:
-      - 560
-      - 560
+      - 518
+      - 518
```

Comment:
@kprokofi Could you explain the reason behind reducing the input size? Also, is dinov2_vits14 backbone the same as dinov2-small-seg?

Reply:
Actually, the image size (518,518) is intended image size for dinov2. DinoV2 is trained on this image size and pos_embeding dim depends on the input resolution.  Current implementation of the VIT backbone in algo/classification will resize pos_embeding to fit input image. It leads to errors, since implementation is a bit different from original facebook repo. Also it may lead to accuracy drop. I tried to avoid resizing, but error in PatchEmbed function is occurred. It is implemented in the timm library, so we would need to bring it into OTX to fix.

Smaller resolution brings better speed and even a better accuracy according to benchmark. 

dinov2_vits14 is the same as dinov2-small-seg. I changed the name to align with other backbones names in the VIT class.